pkg/slices/strings.go

```diff
@@ -2,21 +2,22 @@ package slices
 
 import "strings"
 
+// Union combines to string sets
 func Union(setA, setB []string, ignoreCase bool) []string {
+	exists := Exists
```

Comment:
Ok, I think I see why, but I think I prefer the way it was.  If you wanted to refactor Exists to be a single method with `ignoreCase` as a parameter that might be a better way to handle this.